contracts/03-tamagotchi-nft/tests/owning.rs

```diff
@@ -1,5 +1,6 @@
+use gstd::ActorId;
 use gtest::{Log, Program, System};
-use tamagotchi_interaction_io::{Tamagotchi, TmgAction};
+use tamagotchi_nft_io::{TmgAction, TmgEvent};
 #[test]
 fn smoke_test() {
```

Comment:
The tests also must be written for bad cases:
Not owner is trying to `Transfer`, `Approve` or `RevokeApproval`